tools/ci_build/github/azure-pipelines/templates/rocm.yml

```diff
@@ -0,0 +1,154 @@
+parameters:
+- name: AdditionalBuildFlags
+  type: string
+  default: ''
+
+- name: PythonVersion  
+  type: string
+
+- name: RocmVersion  
+  type: string
+
+
+  
+jobs:
+- job: wheels_python_${{ replace(parameters.PythonVersion,'.','_') }}_rocm_${{ replace(parameters.RocmVersion,'.','_') }}
+  workspace:
+    clean: all
+  timeoutInMinutes: 180
+  pool: Ubuntu-2004-rocm-aiinfra
+  variables:
+    PythonVersion: ${{ parameters.PythonVersion }}
+
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  - template: set-python-manylinux-variables-step.yml
+  - template: get-docker-image-steps.yml
+    parameters:
+      Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_rocm${{ parameters.RocmVersion }}
+      Context: tools/ci_build/github/linux/docker
+      DockerBuildArgs: >-
+        --build-arg TORCH_VERSION=1.11.0
+        --build-arg INSTALL_DEPS_EXTRA_ARGS=-tmur
+        --build-arg BUILD_UID=$(id -u)
+        --network=host --build-arg POLICY=manylinux2014 --build-arg PLATFORM=x86_64
+        --build-arg DEVTOOLSET_ROOTPATH=/opt/rh/devtoolset-10/root
+        --build-arg PREPEND_PATH=/opt/rh/devtoolset-10/root/usr/bin:
+        --build-arg LD_LIBRARY_PATH_ARG=/opt/rh/devtoolset-10/root/usr/lib64:/opt/rh/devtoolset-10/root/usr/lib:/opt/rh/devtoolset-10/root/usr/lib64/dyninst:/opt/rh/devtoolset-10/root/usr/lib/dyninst:/usr/local/lib64:/usr/local/lib
+      Repository: onnxruntimetrainingrocmbuild-rocm${{ parameters.RocmVersion }}
+  - task: CmdLine@2
+    inputs:
+      script: |
+        docker run --rm \
+          --privileged \
+          --ipc=host \
+          --network=host \
+          --cap-add=SYS_PTRACE \
+          --security-opt seccomp=unconfined \
+          -e CC=/opt/rh/devtoolset-10/root/usr/bin/cc -e CXX=/opt/rh/devtoolset-10/root/usr/bin/c++ -e CFLAGS="-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -fstack-clash-protection -fcf-protection -O3 -Wl,--strip-all" -e CXXFLAGS="-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -fstack-clash-protection -fcf-protection -O3 -Wl,--strip-all" \
+          --volume $(Build.SourcesDirectory):/onnxruntime_src \
+          --volume $(Build.BinariesDirectory):/build \
+          --workdir /onnxruntime_src \
+          --entrypoint $(PythonManylinuxDir)/bin/python3 \
+          -e NIGHTLY_BUILD \
+          -e BUILD_BUILDNUMBER \
+          --user onnxruntimedev \
+          onnxruntimetrainingrocmbuild-rocm${{ parameters.RocmVersion }} \
+            /onnxruntime_src/tools/ci_build/build.py \
+              --config Release \
+              --use_rocm \
+              --rocm_version=${{ parameters.RocmVersion }} \
+              --rocm_home=/opt/rocm \
+              --nccl_home=/opt/rocm \
```

Comment:
is the --nccl_home useful here?

Reply:
I don't know. I just keep the original code.